index.html

```diff
@@ -3804,8 +3804,8 @@ <h2><span class="chunk">sBIT</span>
 <p>To simplify decoders, PNG specifies that only certain sample
 depths may be used, and further specifies that sample values
 should be scaled to the full range of possible values at the
-sample depth. The <a class="chunk" href="#11sBIT">
-sBIT</a> chunk defines the original number of
+sample depth. The <span class="chunk">
+sBIT</span> chunk defines the original number of
```

Comment:
In this instance, I removed the link.
This is because it is within the sBIT section it would link to.